backend/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "jsmp-backend",
+  "version": "1.0.0",
+  "description": "javascript mentoring program 2020",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon server/index.ts",
+    "build": "tsc",
+    "test": "jest",
+    "lint": "eslint server --ext 'js,ts'"
+  },
+  "author": "Nataliia Mogylchuk",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nmogylchuk/js-mentoring-program"
+  },
+  "keywords": [
+    "typescript",
+    "node",
+    "express",
+    "nodejs"
+  ],
+  "license": "ISC",
+  "devDependencies": {
+    "@types/express": "^4.17.8",
+    "@types/jest": "^26.0.15",
+    "@types/node": "^14.14.2",
+    "@typescript-eslint/eslint-plugin": "^4.5.0",
+    "@typescript-eslint/parser": "^4.5.0",
+    "eslint": "^7.11.0",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-config-prettier": "^6.14.0",
+    "eslint-loader": "^4.0.2",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-prettier": "^3.1.4",
+    "jest": "^26.6.1",
+    "nodemon": "^2.0.2",
+    "pre-commit": "^1.2.2",
+    "prettier": "2.1.2",
+    "supertest": "^5.0.0",
+    "ts-jest": "^26.4.1",
+    "ts-node": "^9.0.0",
+    "ts-node-dev": "^1.0.0",
+    "tslint": "^6.1.3",
+    "typescript": "^4.0.3"
+  },
+  "dependencies": {
+    "express": "^4.17.1"
+  },
+  "pre-commit": [
```

Comment:
Looks like it doesn't work. Could you try to update pre-commit by this article? https://elijahmanor.com/blog/npm-precommit-scripts ?